lib/propolis/src/vcpu.rs

```diff
@@ -351,12 +355,96 @@ pub mod migrate {
 
     impl BhyveVcpuRunStateV1 {
         fn read(vcpu: &Vcpu) -> io::Result<Self> {
-            let state = vcpu.get_run_state()?;
-            Ok(Self { state: state.state, sipi_vector: state.sipi_vector })
+            let run_state = vcpu.get_run_state()?;
+
+            let vmm_arch: Vec<bhyve_api::vdi_field_entry_v1> =
+                vmm::data::read_many(
+                    vcpu.hdl.as_ref(),
+                    vcpu.id,
+                    bhyve_api::VDC_VMM_ARCH,
+                    1,
+                )?;
+
+            // Load all of the pending interrupt/exception state
+            //
+            // If illumos#15143 support is missing, none of these fields will be
+            // present, so the values will remain false/zeroed.  Such an outcome
+            // is fine for now.
+            let (
+                mut pending_nmi,
+                mut pending_extintr,
```

Comment:
turbo-nit: should this be `pending_extint` to match the struct field?

Reply:
Yeah, I changed the name of the migration data field at some point to match the kernel names, but missed this.

(also, lol @ "turbo-nit")